src/infinite-list/index.tsx

```diff
@@ -162,7 +162,7 @@ const InfiniteList = (props: InfiniteListProps, ref: any) => {
   }, [onScrollBeginDrag, onMomentumScrollEnd, scrollViewProps, isHorizontal]);
 
   const _style = useMemo(() => {
-    return [{height: pageHeight}, style];
+    return {...style, height: pageHeight};
```

Comment:
In this case you're not letting the user override the height prop via style, also if the style is in array there's not need to spread it